bubble_sort.rb

```diff
@@ -0,0 +1,33 @@
+def bubble_sort(array)
+  sorted = false
+  until sorted
+    sorted = true
+    element = 0
+    while element < (array.length - 1)
+      if array[element] > array[element + 1]
+        array[element], array[element + 1] = array[element + 1], array[element]
+        sorted = false
+      end
+      element += 1
+    end
+  end
+  print array
+end
```

Comment:
- [ ] You should be returning `array` not printing it. Remember to use ruby's implicit return when updating your code.